src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -5,7 +5,7 @@ public static class CLIOptions
         public static readonly CLIOption<bool> UseConfigFile = new CLIOption<bool> {
             ArgumentName = "--configFile",
             ArgumentShortName = "-c <useConfigFile>",
-            ArgumentDescription = "Use configFile | Options [true (default), false]",
+            ArgumentDescription = "Use configFile. When not defined you must have a config file called stryker-config.json | Options [true (default), false]",
```

Comment:
Only if config file is used. If using pure commandline run then nog config is needed